.gitignore

```diff
@@ -28,6 +28,7 @@ venv
 .coverage
 .tox
 tests/output
+neurodamus-models*
```

Comment:
No more needed after https://github.com/openbraininstitute/neurodamus/pull/12 moves the downloading of `neurodamus-models` in `.tox`.